test/test_action_controller.rb

```diff
@@ -0,0 +1,85 @@
+require 'test/unit'
+require 'mocha/test_unit'
+require 'active_support'
+require 'action_controller'
```

Comment:
The tests are failing on this require. You mentioned that this PR would fail until the netuitive_ruby_api is deployed, but that would only affect the `require 'netuitive/*'` imports below, not this, correct?